cross/rutorrent/patches/001-expanding-external-programs.patch

```diff
@@ -0,0 +1,15 @@
+Added entries corresponding to external programs which are also potentially used by plugins and
+may therefore be configured
+index b68c526c..604f6263 100644
+--- conf/config.php.orig
++++ conf/config.php
+@@ -52,6 +52,8 @@
+ 		"gzip"	=> '',			// Something like /usr/bin/gzip. If empty, will be found in PATH.
+ 		"id"	=> '',			// Something like /usr/bin/id. If empty, will be found in PATH.
+ 		"stat"	=> '',			// Something like /usr/bin/stat. If empty, will be found in PATH.
++		"python"=> '',			// Something like /usr/bin/python3. If empty, will be found in PATH
++		"pgrep"	=> '',			// Something like /usr/bin/pgrep. If empty, will be found in PATH 
```

Comment:
This is to be able to define explicitely the location where to look for pgrep binary.